svc.html

```diff
@@ -0,0 +1,347 @@
+    to be able to decode anything that an encoder can send so that configuration is not required
+    to enable an <code><a>RTCRtpReceiver</a></code> to decode scalable video coding.</p> 
+  </section>
+  <section id="rtpdictionaries">
+    <h2>Dictionary extensions</h2>
+    <section id="rtcrtpencodingparameters">
+      <h3>RTCRtpEncodingParameters Dictionary Extensions</h3>
+      <div>
+        <pre class="idl">partial dictionary RTCRtpEncodingParameters {
+             double              scaleFramerateDownBy;
+             DOMString           encodingId;
+             sequence&lt;DOMString&gt; dependencyEncodingIds;
+};</pre>
+        <section>
+          <h2>Dictionary <a class="idlType">RTCRtpEncodingParameters</a> Members</h2>
+          <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for=
+          "RTCRtpEncodingParameters" class="dictionary-members">
+            <dt><dfn data-idl><code>scaleFramerateDownBy</code></dfn> of type <span class=
+            "idlMemberType"><a>double</a></span></dt>
+            <dd>
+              <p>Inverse of the input framerate fraction to be encoded. Example: 1.0 =
+              full framerate, 2.0 = one half of the full framerate. For scalable video
+              coding, <code>scaleFramerateDownBy</code> refers to the inverse of the aggregate
+              fraction of input framerate achieved by this layer when combined with all
+              dependent layers.</p>
+            </dd>
+            <dt><dfn data-idl><code>encodingId</code></dfn> of type <span class=
+            "idlMemberType"><a>DOMString</a></span></dt>
+            <dd>
+              <p>An identifier for the encoding object. This identifier should be unique
+              within the scope of the localized sequence of
+              <code><a>RTCRtpEncodingParameters</a></code> for any given
+              <code><a>RTCRtpParameters</a></code> object. Values MUST be composed only
+              of alphanumeric characters (a-z, A-Z, 0-9) up to a maximum
+              of 16 characters. While the <code>encodingId</code> attribute has the same syntax
+              as the <code>rid</code> attribute, these attributes are distinct, since the
+              <code>encodingId</code> attribute does not imply inclusion of a RID header extension
+              within RTP packets.</p>
+            </dd>
+            <dt><dfn data-idl><code>dependencyEncodingIds</code></dfn> of type <span class=
+            "idlMemberType">sequence&lt;<a>DOMString</a>&gt;</span></dt>
+            <dd>
+              <p>The <code><a>encodingId</a></code>s on which this layer depends. Within
+              this specification <code><a>encodingId</a></code>s are permitted only
+              within the same <code><a>RTCRtpEncodingParameters</a></code> sequence.
+              In order to send scalable video coding
+              (<a>SVC</a>), both the <code>encodingId</code> and
+              <code>dependencyEncodingIds</code> are required.</p>
```

Comment:
What does it mean that one layer depends on another? Can you create invalid dependencies?

Reply:
Yes, quite easily. VP9 has a fixed dependency structure where each temporal or spatial layer depends only on the immediate lower one.

```
   Within a picture, a frame with spatial layer ID equal to SID, where
   SID > 0, can depend on a frame of the same picture with a lower
   spatial layer ID.  This "inter-layer" dependency can result in
   additional coding gain compared to the case where only traditional
   "inter-picture" dependency is used, where a frame depends on
   previously coded frame in time.  For simplicity, this payload format
   assumes that, within a picture and if inter-layer dependency is used,
   a spatial layer SID frame can depend only on the immediately previous
   spatial layer SID-1 frame, when S > 0.  Additionally, if inter-
   picture dependency is used, a spatial layer SID frame is assumed to
   only depend on a previously coded spatial layer SID frame
```

Reply:
@pthatcher An alternative is to enable a choice between “canned” scalability structures, such as those described in AV1 Section 6.7.5. This limits the ability to specify invalid structures and could also support K-SVC.

Reply:
@henbos The proposal has been updated to utilize "scalability modes".  This limits the ability to specify invalid scalability dependencies.